script.js

```diff
@@ -0,0 +1,68 @@
+const calcInput = document.querySelector("#calc-input")
+const lineOne = document.querySelector("#line-one")
+const lineTwo = document.querySelector("#line-two")
+const lineThree = document.querySelector("#line-three")
+const lineFour = document.querySelector("#line-four")
+const plusButton = document.querySelector("#plus-button")
+const minusButton = document.querySelector("#minus-button")
+const multiplyButton = document.querySelector("#multiply-button")
+const divideButton = document.querySelector("#divide-button")
+const resultButton = document.querySelector("#result-button")
+const clearButton = document.querySelector("#clear-button")
+
+let inputType = typeof(Number(calcInput.value))
```

Comment:
no need to write brackets after typeof [link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof)